### ISSUE TITLE
XXX: number is not absolutely safe for someone

### ISSUE THREAD

**Issue by urain39** [urain39/ij2tpl.js#231]
https://github.com/urain39/ij2tpl.js/blob/0fa3aed397d3deff8eb49af5a969c83b5f2979e5/ij2tpl.ts#L449

Number is not absolutely safe for someone, if they want to escape numbers.

**Comment by urain39** [comment]
```js
IJ2TPL.setEscape((value) => value.replace(/(\d+)/g), '\\$1', (value) => typeof value === 'number');
```

**Comment by urain39** [comment]
```js
IJ2TPL.setEscape(
    (value) => value.replace(/(\d)/g), '\\$1'),
    (value) => typeof value === 'number'
);
```

**Comment by urain39** [comment]
```js
IJ2TPL.setEscape(
    (value) => value.replace(/(\d)/g, '\\$1'),
    (value) => typeof value === 'number'
);
```

**Comment by urain39** [comment]
```js
IJ2TPL.setEscapeFunction(result => result);
```

**Comment by urain39** [comment]
Fixed, see #237
